### PR TITLE
[SKY30-201] Fix details table headers on scroll

### PR DESCRIPTION
**Pull request by SARodrigues**
### Overview

This PR fixes the details tables' header to the top on scroll. 

**Note:** 
It looks like `position: sticky` doesn't play very well when parent containers have an overflow set, so there was some trickery involved in order to make this work. The headers' borders wouldn't display on scroll either, hence the extra global. 

### Feature relevant tickets

[SKY30-201](https://vizzuality.atlassian.net/browse/SKY30-201)

[SKY30-201]: https://vizzuality.atlassian.net/browse/SKY30-201?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: frontend/src/styles/globals.css [hunk 1]
```diff
@@ -38,3 +38,6 @@
   @apply !border-r-black
 }
 
+.details-table-first-column-border {
```
Comment (andresgnlez) [inline frontend/src/styles/globals.css:41]: Why are you declaring a class here?

File: frontend/tailwind.config.js [hunk 2]
```diff
@@ -49,6 +49,10 @@ module.exports = {
         'accordion-down': 'accordion-down 0.2s ease-out',
         'accordion-up': 'accordion-up 0.2s ease-out',
       },
+      borders: {
+        test: 'border-right-style: dashed',
+        r: { test: 'border-right-style: dashed' },
+      },
```
Comment (andresgnlez) [inline frontend/tailwind.config.js:55]: Are you using these?

**Review by andresgnlez — CHANGES_REQUESTED** [review]

File: frontend/tailwind.config.js (hunk 2 above)
Comment (SARodrigues) [reply frontend/tailwind.config.js:55]: Uff it looks like I have pushed my test branch, not the correct one. I'll take a look.

File: frontend/src/styles/globals.css (hunk 1 above)
Comment (SARodrigues) [reply frontend/src/styles/globals.css:41]: Uff it looks like I have pushed my test branch, not the correct one. I'll take a look.

File: frontend/tailwind.config.js (hunk 2 above)
Comment (SARodrigues) [reply frontend/tailwind.config.js]: Test config removed.

File: frontend/src/styles/globals.css (hunk 1 above)
Comment (SARodrigues) [reply frontend/src/styles/globals.css:41]: @andresgnlez 

In these first column cells, the bottom border is solid (much like in all cells), but in these particular ones the right border needs to be dashed. I haven't found a way to pull this off the regular tailwind way, hence the custom style. 

Let me know if you know of a way.

**Comment by andresgnlez** [comment]
@SARodrigues let me know if my last commit works for you.

**Comment by SARodrigues** [comment]
@andresgnlez LGTM, thank you. The margin on the right side of the table is a bit off but we can address it afterwards

**Pull request merged**